src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  width: $card-width;
+
+  padding: 32px 16px 16px;
+
+  background-color: $color-white;
+  border: 1px solid $color-concrete;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+
+    object-fit: contain;
+  }
+
+  &__heading {
+    @extend %text-reset;
+
+    margin-bottom: 4px;
+
+    color: $color-accent;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
```

Comment:
It would be better to use variables for all this values(font-size, font-weight, line-height)

Reply:
i have various of values in font-weight and etc.